### PR TITLE
[#2] Add Cardano.Prelude

### PR DESCRIPTION
**Pull request by ruhatch**

### CODE REVIEW

**Review by redxaxder — APPROVED** [review]
Looks good to me

File: prelude/stack.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+resolver: lts-12.10
```
Comment (dnadales) [inline prelude/stack.yaml]: Shouldn't we have the `stack.yaml` at the top level? I guess `prelude` will be used by the other packages in this repo, won't it?

Comment (nc6) [reply prelude/stack.yaml]: Indeed, there's no point having a stack.yaml file below the top level, it would just be ignored.

Comment (dnadales) [reply prelude/stack.yaml]: @ruhatch pointed out that we might want to factor our some packages to different repositories. However, we can do that when that time comes.

**Pull request merged**